athena/llm_core/llm_core/utils/llm_utils.py

```
@@ -66,45 +65,30 @@
     return prompt_input, False
 
 
-def supports_function_calling(model: BaseLanguageModel):
-    """Returns True if the model supports function calling, False otherwise
+def get_chat_prompt(
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)